linera-service/src/linera/main.rs

```diff
@@ -1251,17 +1251,17 @@ impl Runnable for Job {
                     {faucet_url}",
                 );
                 let faucet = cli_wrappers::Faucet::new(faucet_url);
-                let outcome = faucet.claim(&owner).await?;
-                println!("{}", outcome.chain_id);
-                println!("{}", outcome.certificate_hash);
-                println!("{}", owner);
+                let description = faucet.claim(&owner).await?;
+                println!("{}", description.id());
+                serde_json::to_writer(std::io::stdout(), &description)?;
```

Comment:
Maybe that doesn't need to be on `stdout`, and should be a debug or info log instead?

Reply:
Oh I thought I was going to need it, but I don't really.  I'll remove it entirely.